bin/kmigrator.py

```diff
@@ -49,8 +49,10 @@
 NAME = '_django_model_generator'
 MODELS_TPL = """
 from django.db import models
```

Comment:
That import doesn't seem to be needed. You have 53 line, where you are imported all from `models` package. So you can replace `models.Model` to simply `Model` call.

Reply:
It’s required for backward compatibility